icu4j/main/classes/core/src/com/ibm/icu/text/DecimalFormat.java

```diff
@@ -2194,6 +2194,14 @@ public synchronized void setParseStrict(boolean parseStrict) {
     refreshFormatter();
   }
 
+  /**
+   * @internal
```

Comment:
Can you add a little documentation here about why you might want to use this method (even though it's internal)?

Reply:
Done. PTAL